llo/aggregators.go

```diff
@@ -53,8 +52,65 @@ func MedianAggregator(values []StreamValue, f int) (StreamValue, error) {
 	return ToDecimal(observations[len(observations)/2]), nil
 }
 
+// ModeAggregator works on arbitrary StreamValue types
+// It picks the most common value
+// There must be at least f+1 observations in agreement in order to produce a value
+// nil observations are ignored
 func ModeAggregator(values []StreamValue, f int) (StreamValue, error) {
-	return nil, errors.New("not implemented")
+	// remove nils
```

Comment:
Do we need to check for empty an empty slice here? Or this will never get called with a zero or nil []StreamValue?